### PR TITLE
Make the Atom feed valid

### PR DESCRIPTION
**Pull request by asbjornu**
This PR does a few things to ensure that the [currently invalid Atom feed](https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fwww.rabbitmq.com%2Fnews.atom) is made valid. The feed started out with the following list of warnings:

```sh
$ python src/demo.py http://localhost:8191/news.atom
Validating http://localhost:8191/news.atom
line 2, column 0: Use of unknown namespace: http://www.rabbitmq.com/namespaces/ad-hoc/doc
line 2, column 0: Avoid Namespace Prefix: atom
line 2, column 0: Avoid Namespace Prefix: xhtml
line 2, column 168: updated must be an RFC-3339 date-time
line 2, column 0: Missing feed element: title
line 2, column 0: Missing feed element: id
line 2, column 0: Missing atom:link with rel="self"
line 2, column 342: updated must be an RFC-3339 date-time (57 occurrences)
line 29, column 33: Missing entry element: author (100 occurrences)
line 1993, column 13: p contains bad characters (2 occurrences)
```

Which should all be gone after merging this and doing `xsltproc site/feed-atom.xsl site/news.xml > site/news.atom` to generate a new `news.atom` feed.

### CODE REVIEW

**Comment by asbjornu** [comment]
Sorry, I forgot to squash before I pushed. Closing this and will open a new, squashed PR.

**Comment by michaelklishin** [comment]
@asbjornu thank you! FTR, we don't really mind non-squared PRs.

File: site/feed-atom.xsl [hunk 1]
```diff
@@ -27,48 +28,62 @@ limitations under the License.
...
-      <atom:updated><xsl:value-of select="doc:date/@iso"/></atom:updated>
-      <atom:content type="xhtml">
-        <xhtml:div>
+      <id>tag:rabbitmq.net,2007:<xsl:value-of select="doc:date/@iso"/></id>
+      <title type="text"><xsl:value-of select="doc:title"/></title>
+      <updated><xsl:value-of select="doc:date/@iso"/></updated>
+      <author>
+          <name>RabbitMQ</name>
+          <uri>https://www.rabbitmq.com/</uri>
+          <email>support@rabbitmq.com</email>
```
Comment (michaelklishin) [inline site/feed-atom.xsl]: Please change this to `info@rabbitmq.com`.

Comment (asbjornu) [inline site/feed-atom.xsl]: Done in #121

**Comment by asbjornu** [comment]
@michaelklishin Thanks! A squashed and fixed #121 submitted anyway. :)

**Pull request closed without merging**